pages/bare_metal_cloud/dedicated_servers/ola-enable-debian12-netplan/guide.en-gb.md

```diff
@@ -0,0 +1,175 @@
+
+```bash
+vi /etc/netplan# cat 50-bond.yaml
+```
+
+To configure the bond interface, insert the following configuration:
+
+```bash
+network: 
+    version: 2
+    ethernets: 
+        ens22f0np0:
+            match: 
+                macaddress: a1:b2:c3:d4:e5:c6
+        ens22f1np1:
+            match: 
+                macaddress: a1:b2:c3:d4:e5:c7
+    bonds: 
+        bond0:
+            # MAC address of the server's main interface
+            macaddress: a1:b2:c3:d4:e5:c6
+            accept-ra: false
+            addresses: 
+                - 141.94.154.196/32
+                - 2001:41d0:254:c400::/56
+            routes: 
+                - on-link: true
+                  to: default
+                  via: 100.64.0.1
+                - on-link: true
+                  to: default
+                  via: fe80::1
+            nameservers: 
+                addresses: 
+                - 2001:41d0:3:163::1
+            interfaces: 
+                - ens22f0np0
+                - ens22f1np1
+            parameters: 
+                mode: 802.3ad
+                mii-monitor-interval: 100
+                down-delay: 200
+                lacp-rate: slow
+                transmit-hash-policy: layer3+4
+```
+
+### DHCP configuration
+
+Create the interfaces file in a text editor of your choice using the following command:
+
+```bash
+vi /etc/netplan# cat 50-cloud-init.yaml_dhcp
+```
+
+To configure the bond interface, insert the following configuration:
+
+```bash
+network: 
+    version: 2
+    ethernets: 
+        ens22f0np0:
+            match: 
+                macaddress: a1:b2:c3:d4:e5:c6
+        ens22f1np1:
+            match: 
+                macaddress: a1:b2:c3:d4:e5:c7
+    bonds: 
+        bond0:
+            # MAC address of the server's main interface
+            macaddress: a1:b2:c3:d4:e5:c6
+            accept-ra: false
+            dhcp4: true
+            addresses: 
+                - 2001:41d0:254:c400::/56
+            routes: 
+                - on-link: true
+                  to: default
+                  via: fe80::1
+            nameservers: 
+                addresses: 
+                - 2001:41d0:3:163::1
+            interfaces: 
+                - ens22f0np0
+                - ens22f1np1
+            parameters: 
+                mode: 802.3ad
+                mii-monitor-interval: 100
+                down-delay: 200
+                lacp-rate: slow
+                transmit-hash-policy: layer3+4
+```
+
+### Testing and applying the configuration
+
+You can test your configuration using this command:
+
+```bash
+sudo netplan try
+```
+
+If it is correct, apply it using the following command:
+
+```bash
+sudo netplan apply
+```
+
+This restart may take several minutes since it is building the bond interface.  To test that our bond is working, ping another server on the same vRack. If it works, you are all set. If it does not, double check your configurations or try rebooting the server.
```

Comment:
```suggestion
It may take several seconds for the bond interface to come up.
```
J'enlève le reste car un reboot n'aidera pas ici. Et on parle pas de vRack.